mypy/semanal.py

```diff
@@ -1855,8 +1862,14 @@ def add_method(funcname: str,
 
         add_method('_replace', ret=selftype,
                    args=[Argument(var, var.type, EllipsisExpr(), ARG_NAMED_OPT) for var in vars])
+
+        def make_init_arg(var: Var) -> Argument:
+            default = default_items.get(var.name(), None)
```

Comment:
The default value could be itself ``None``. So that I would use ``sentinel = object()`` and then ``default_items.get(var.name(), sentinel)`` and below ``if default is sentinel``.